src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  --bg-color: #fff;
+  --txt-color: #000;
+  --txt-color-a: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: var(--bg-color);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
```

Comment:
We can write like so 
```
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```
It means that every item has 20 px margin left except the first one